### PR TITLE
Enhance frictional translational models based on ExternalCombiTable1D

### LINKED ISSUE #3664: Improve design of frictional translational components to use static tables instead of dynamic
Changes suggested in #3661 and realised in PR #3662 shall be applied accordingly to translational components:
- SupportFriction
- Brake

### PR DESCRIPTION
**Pull request by tobolar**
See also #3662
Close #3664

### CODE REVIEW

File: Modelica/Mechanics/Translational/Components/Brake.mo [hunk 1]
```diff
@@ -1,43 +1,62 @@
...
+    extrapolation=Extrapolation.LastTwoPoints,
+    verboseRead=false) "External table object for sliding friction coefficient";
+  final parameter Real mu0=
+    if     smoothness == Smoothness.ConstantSegments then getTable1DValueNoDer(tableID, 1, 0)
+    elseif smoothness == Smoothness.LinearSegments   then getTable1DValueNoDer2(tableID, 1, 0)
+    else                                                  getTable1DValue(tableID, 1, 0)
+    "Friction coefficient for w=0 and forward sliding";
+
+  Real table_signs[2]
+    "Signs for sliding friction coefficient table interpolation: [sign for w_rel, sign for mu]";
```
Comment (beutlich) [inline Modelica/Mechanics/Translational/Components/Brake.mo:51]: There is no `w_rel` in Brake.

File: Modelica/Mechanics/Translational/Components/SupportFriction.mo [hunk 2]
```diff
@@ -1,51 +1,71 @@
...
+    tableName="NoName",
+    fileName="NoName",
+    table=f_pos,
+    columns={2},
+    smoothness=smoothness,
+    extrapolation=Extrapolation.LastTwoPoints,
+    verboseRead=false) "External table object for sliding friction coefficient";
+
+  Real table_signs[2]
+    "Signs for sliding friction coefficient table interpolation: [sign for w_rel, sign for mu]";
```
Comment (beutlich) [inline Modelica/Mechanics/Translational/Components/SupportFriction.mo:34]: There is neither `w_rel` nor `mu` in SupportFriction.

File: Modelica/Mechanics/Translational/Components/Brake.mo (hunk 1 above)
Comment (tobolar) [reply Modelica/Mechanics/Translational/Components/Brake.mo]: Fix with 850a094b505c4e4737beccd24ef8a4ce6f85f114

File: Modelica/Mechanics/Translational/Components/SupportFriction.mo (hunk 2 above)
Comment (tobolar) [reply Modelica/Mechanics/Translational/Components/SupportFriction.mo]: Fix with 9124e3f561ba0f418ba64028ed0c56e05157f778

**Review by MartinOtter — APPROVED** [review]

File: Modelica/Mechanics/Translational/Components/Brake.mo [hunk 3]
```diff
@@ -1,75 +1,85 @@
...
-
   extends Modelica.Mechanics.Translational.Interfaces.PartialElementaryTwoFlangesAndSupport2;
   extends Modelica.Thermal.HeatTransfer.Interfaces.PartialElementaryConditionalHeatPortWithoutT;
+
+  import Modelica.Blocks.Types.{ExternalCombiTable1D, Extrapolation, Smoothness};
+  import Modelica.Blocks.Tables.Internal.{getTable1DValue, getTable1DValueNoDer, getTable1DValueNoDer2};
+
   parameter Real mu_pos[:, 2]=[0, 0.5]
-    "Positive sliding friction coefficient [-] as function of v_rel [m/s] (v_rel>=0)";
+    "Positive sliding friction coefficient [-] as function of v [m/s] (v>=0)";
```
Comment (HansOlsson) [inline Modelica/Mechanics/Translational/Components/Brake.mo:10]: Why is this description string oscillating back and forth?
In MSL 3.2.3 it was `v` - and this indicates that it was changed to `v_rel` and is now changed back to `v`.

File: Modelica/Mechanics/Translational/Components/SupportFriction.mo [hunk 4]
```diff
@@ -1,51 +1,71 @@
...
-  f = if locked then sa*unitForce else (
-    if startForward then
-      Modelica.Math.Vectors.interpolate(f_pos[:, 1], f_pos[:, 2], v, 1)
-    else if startBackward then
-      -Modelica.Math.Vectors.interpolate(f_pos[:, 1], f_pos[:, 2], -v, 1)
-    else if pre(mode) == Forward then
-      Modelica.Math.Vectors.interpolate(f_pos[:, 1], f_pos[:, 2], v, 1)
-    else
-      -Modelica.Math.Vectors.interpolate(f_pos[:, 1], f_pos[:, 2], -v, 1));
+  table_signs =
```
Comment (HansOlsson) [inline Modelica/Mechanics/Translational/Components/SupportFriction.mo:60]: Ok. However, be careful with such rewriting as some things that look harmless might introduce chattering in rare cases.

**Review by HansOlsson — APPROVED** [review]
Looks good.

File: Modelica/Mechanics/Translational/Components/Brake.mo (hunk 3 above)
Comment (tobolar) [reply Modelica/Mechanics/Translational/Components/Brake.mo:10]: Actually, the model itself contains no such variable. Either `v` or `v_relfric`. There is used `v` in the model's documentation, so this shall be the same for consistency.

File: Modelica/Mechanics/Translational/Components/SupportFriction.mo (hunk 4 above)
Comment (tobolar) [reply Modelica/Mechanics/Translational/Components/SupportFriction.mo:60]: This is principally similar to `Mechanics.Rotational.Components.BearingFriction`, see https://github.com/modelica/ModelicaStandardLibrary/pull/3662 (already merged).

File: Modelica/Mechanics/Translational/Components/Brake.mo (hunk 3 above)
Comment (beutlich) [reply Modelica/Mechanics/Translational/Components/Brake.mo:10]: See also #3694.

**Pull request merged**